contracts/GenericBondCalculator.sol

```diff
@@ -1,93 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
Why is all this logic no longer needed?

Reply:
Generic bond calculator used the old IDF approach, that is no longer the case. The new approach requires a new set of functions.